Modules/TPC/src/Clusters.cxx

```diff
@@ -113,7 +130,31 @@ void Clusters::reset()
 {
   // clean all the monitor objects here
 
-  QcInfoLogger::GetInstance() << "Resetting the histogram" << AliceO2::InfoLogger::InfoLogger::endm;
+  QcInfoLogger::GetInstance() << "Resetting the canvases" << AliceO2::InfoLogger::InfoLogger::endm;
+
```

Comment:
Also this is quite some copy and paste. Perhaps this could also be simplified at some point.

Reply:
I added `clearCanvases`.